### ISSUE TITLE
Update documentation with custom binary path information

### ISSUE THREAD

**Issue by jsfehler** [cobrateam/splinter#811]
ie: Update https://splinter.readthedocs.io/en/latest/drivers/chrome.html with

```
from selenium import webdriver

chrome_options = webdriver.chrome.options.Options()
chrome_options.binary_location = "/path/to/canary"
browser = Browser('chrome', options=chrome_options)
```

but for Firefox as well.

**Comment by ketangupta34** [comment]
I would like to work on this!

**Comment by jsfehler** [comment]
@ketangupta34  Go for it

**Comment by mohitdmak** [comment]
Is this issue open?

**Comment by andrewsmedina** [comment]
@mohitdmak yes

**Comment by mohitdmak** [comment]
ok, so do I need to be assigned this to work on it?

**Comment by andrewsmedina** [comment]
@mohitdmak the assign is not necessary. a message here informing that you are working on this issue is enough.

**Comment by arshm06** [comment]
I have submitted a pull request. Thank you!

**Comment by jsfehler** [comment]
Addressed by #1184
